codegen/golang.go

```diff
@@ -87,6 +87,13 @@ func WithUnwrapAllEmbeddedStructs() TableOptions {
 	}
 }
 
+// Allows overriding the schema type for a specific field. Return `nil` to fallback to default behavior
+func WithValueTypeOverride(resolver func(reflect.StructField) *schema.ValueType) TableOptions {
```

Comment:
Let's change it to standard godoc style
```suggestion
// WithValueTypeOverride sets a function that can override the schema type for specific fields. Return `nil` to fall back to default behavior.
func WithValueTypeOverride(resolver func(reflect.StructField) *schema.ValueType) TableOptions {
```

Reply:
(We should probably apply this consistently everywhere, but good to start somewhere...)